### DISCUSSION TITLE
Pattern Variables included from commands - Detailed output and reference.

### DISCUSSION THREAD

**Discussion by technotion** [danielmiessler/Fabric#786] [Ideas]
I've been using Fabric setup to output to an Obsidian vault, or more specifically the `| save` function to a .md file. I've been wondering if it is possible to pass a variable from the Fabric command into the pattern output. e.g. the title and URL of the YouTube video that was used in the command.

**Example setup:**
`yt --transcript https://www.youtube.com/watch?v=qs2QcycggWU | fabric --model claude-3-5-sonnet-20240620 -sp extract_wisdom | save "starlink"`

The output of the .md file. from the 'extract_wisdom' pattern.

---
SUMMARY
Title of content: H1 Title
URL of Content: https://url.url
The short description....
---

The thought is that when saving to the .md file for note taking, the output provides a reference for what derived the content from the pattern.

**Comment by JuracyAmerico** [comment]
@technotion, great idea.
 This would be very useful, especially for tracking all the academic reference resources.
